jumpstart-launch-tasks.sh

```diff
@@ -2,10 +2,12 @@
 read -p "Enter the site shortname on ACSF (e.g., \"jumpstartlockss\", no quotes): " sitename
 # Baby, what time is it?
 timestamp=$(date +%Y%m%d%H%M%S)
-howardpass="cat /dev/urandom | tr -cd a-zA-Z0-9_+=*\ | fold -w18 | head -n 1"
-lindseypass="cat /dev/urandom | tr -cd a-zA-Z0-9_+=*\ | fold -w18 | head -n 1"
-richardpass="cat /dev/urandom | tr -cd a-zA-Z0-9_+=*\ | fold -w18 | head -n 1"
-adminpass="cat /dev/urandom | tr -cd a-zA-Z0-9_+=*\ | fold -w18 | head -n 1"
+# Generate a random 18-character password that can include mixed-case,
+# alphanumeric, and "_-!.+".
+howardpass=$(cat /dev/urandom | LC_ALL=C tr -cd a-zA-Z0-9\_\-\!\.\+ | fold -w18 | head -n 1)
```

Comment:
Same as above. 

Tried it in an ad-hoc bash script and got same results. 

![Screen Shot 2021-06-02 at 3 16 53 PM](https://user-images.githubusercontent.com/8387457/120553303-90c10d80-c3b5-11eb-930f-848f4e4f15c6.png)

Reply:
I don't think we need this script anymore. I'll keep it in this repo, but not bother modifying it.